specification/schemaregistry/SchemaRegistry/tspconfig.yaml

```diff
@@ -0,0 +1,39 @@
+parameters:  
+  "service-dir":
+    default: "sdk/schemaregistry"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/../data-plane/SchemaRegistry/preview/2023-07-01"
```

Comment:
This needs to point to the stable/ dir.

Reply:
sounds good! will update this to use {version_status}/{version}